.github/workflows/snap.yaml

```diff
@@ -8,18 +8,23 @@ on:
       - noetic-devel
   workflow_dispatch:
   workflow_call:
+   inputs:
+      branch-name:
+        required: false
+        type: string
+        default: '${{ github.ref }}'
 
 jobs:
   snap:
-    uses: ubuntu-robotics/snap_workflows/.github/workflows/snap.yaml@main
+    uses: ubuntu-robotics/snap_workflows/.github/workflows/snap.yaml@fix/default_branch_name
     secrets:
       store-login: ${{ secrets.STORE_LOGIN }}
     with:
       ubuntu-image: ubuntu-20.04
-      branch-name: noetic-devel
+      branch-name: ${{ inputs.branch-name }}
```

Comment:
Isn't this input variable only available on `workflow_call`?

Reply:
Indeed, I will use an if condition instead.